### PR TITLE
feat(navigation): replace top-toolbar board picker with bottom-left FAB cluster

### PR DESCRIPTION
**Pull request by OPS-PIvers**
## Summary
- Removes the chevron-expanded pill picker from the top floating toolbar
- Adds a 3-button FAB cluster in the bottom-left: `[ ◀ ] [ ⋮ ] [ ▶ ]` styled to match the existing help and background-music FABs
- Prev/next wrap around the dashboard list (matches existing Alt+←/→ behavior)
- The kebab opens a glass-styled popover above the cluster with the full board list — supports keyboard nav (Escape closes), click-outside to dismiss, ⭐ for default board, active row highlighted
- Cluster auto-hides when there is only one board
- Music/help FABs now stack on top of the nav cluster on the left side when they would otherwise overlap (e.g., right-side dock or video background)
- Updates the `switchBoardsHint` empty-state copy in all four locales (en/de/es/fr) and adds new `boardNav.*` keys

## Files
- `components/layout/BoardNavFab.tsx` (new) — cluster + popover
- `components/layout/DashboardView.tsx` — mounts `BoardNavFab`, adjusts music & help FAB stacking
- `components/layout/sidebar/Sidebar.tsx` — removes the chevron + horizontal pill bar and the now-orphaned state/refs/effects (`isBoardSwitcherExpanded`, `scrollContainerRef`, `toolbarRef`, `canScrollRight`, `checkScroll`)
- `locales/{en,de,es,fr}.json` — adds `boardNav.*` keys, updates `switchBoardsHint` copy

## Test plan
- [ ] Multi-board dashboard: click ◀ / ▶ to cycle; verify slide animation triggers in correct direction
- [ ] Cycling wraps at both ends
- [ ] Kebab opens popover above cluster; correct active row highlighted; ⭐ shows for default board
- [ ] Click-outside and Escape both close the popover
- [ ] Single-board dashboard: cluster does not render
- [ ] Default dock (`bottom`): cluster bottom-left, help bottom-right; with video bg, music stacks above cluster
- [ ] Dock `left`: cluster bottom-left (lives below the vertical dock rail), help bottom-right
- [ ] Dock `right`: cluster bottom-left, help on the left stacked above music+cluster
- [ ] Top toolbar no longer shows the chevron / expanded pill picker
- [ ] Existing Alt+←/→ keyboard shortcut and 2-finger swipe still work
- [ ] Empty-board hint copy reads correctly in all 4 locales

https://claude.ai/code/session_015NhosBSSRhEnvYCUD9nT9q

---
_Generated by [Claude Code](https://claude.ai/code/session_015NhosBSSRhEnvYCUD9nT9q)_

### CODE REVIEW

File: components/layout/DashboardView.tsx [hunk 1]
```diff
@@ -1446,8 +1450,8 @@ export const DashboardView: React.FC = () => {
               ? 'Enable background video sound'
               : 'Mute background video'
           }
-          className={`fixed bottom-6 z-dock w-8 h-8 rounded-full bg-white/10 hover:bg-white/20 border border-white/20 text-white/60 hover:text-white/90 flex items-center justify-center transition-colors backdrop-blur-sm ${
-            dockPosition === 'left' ? 'right-14' : 'left-4'
+          className={`fixed left-4 z-dock w-8 h-8 rounded-full bg-white/10 hover:bg-white/20 border border-white/20 text-white/60 hover:text-white/90 flex items-center justify-center transition-colors backdrop-blur-sm ${
+            dashboards.length > 1 ? 'bottom-16' : 'bottom-6'
           }`}
```
[the remark this answers is not in the document]
Comment (OPS-PIvers) [reply components/layout/DashboardView.tsx:1455]: Skipping intentionally — the placement here was chosen deliberately by the PR author. The new `BoardNavFab` cluster sits at bottom-left in all dock configurations, and the music FAB stacks vertically above it so they share a single edge rather than splitting controls across both sides. Restoring the dock-conditional flip would put the music FAB on the right side only when the dock is on the left, which would split the on-screen FABs across both edges depending on dock position and undermine the consistency the cluster's fixed placement is meant to provide. Happy to revisit holistic FAB placement in a dedicated PR if the team wants to move toward a unified opposite-side rule across all overlays.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: components/layout/BoardNavFab.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                key={db.id}
+                role="option"
+                aria-selected={isActive}
+                onClick={() => {
+                  loadDashboard(db.id);
+                  setIsPickerOpen(false);
+                }}
+                className={`w-full flex items-center gap-2 px-3 py-2 text-left text-sm transition-colors ${
+                  isActive
+                    ? 'bg-brand-blue-primary text-white'
```
[the remark this answers is not in the document]
Comment (OPS-PIvers) [reply components/layout/BoardNavFab.tsx:151]: Already addressed on this branch. `BoardNavFab.tsx` switched from `role="listbox"` to `role="menu"` (which is the more honest pattern for a simple button list), focus is moved into the active board on open via the effect at lines 48–52, focus is restored to the trigger on close via `closePicker(returnFocus = true)` at line 36, and a full keyboard handler (Up/Down/Home/End/Escape/Tab) is wired up at lines 80–110 with arrow keys driving roving focus through `itemRefs.current[idx].focus()`. No additional code change needed.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: components/layout/BoardNavFab.tsx [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+            {boardListLabel}
+          </div>
+          {dashboards.map((db, idx) => {
+            const isActive = activeDashboard?.id === db.id;
+            return (
+              <button
+                key={db.id}
+                ref={(el) => {
+                  itemRefs.current[idx] = el;
+                }}
```
[the remark this answers is not in the document]
Comment (OPS-PIvers) [reply components/layout/BoardNavFab.tsx:143]: Already addressed on this branch via a different mechanism. Rather than `splice` inside the ref callback, `BoardNavFab.tsx` lines 56–58 truncate `itemRefs.current.length = dashboards.length` in an effect keyed on `dashboards.length`, which prevents stale refs to deleted boards from accumulating. Equivalent outcome (no focus dispatched to detached buttons after deletion) and slightly simpler than splicing per-callback. No additional code change needed.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by OPS-PIvers — COMMENTED** [review]
## Automated Code Review

**Date:** 2026-05-01

### Summary
Replaces the chevron-expanded board picker that lived in the top-floating `Sidebar` toolbar with a 3-button FAB cluster at bottom-left (`◀ ⋮ ▶`), matching the visual language of the existing help / background-music FABs. Adds the new `BoardNavFab` component (209 lines), refactors the help-FAB's positioning logic in `DashboardView.tsx` from a nested ternary into a named-variable IIFE, removes the now-unused board-switcher state/refs/effects from `Sidebar.tsx`, and updates the `switchBoardsHint` empty-state copy in all four locales. The cluster auto-hides when there's only one board, and supports keyboard navigation (↑/↓/Home/End/Esc/Tab) with focus restoration to the trigger on close.

### Code Quality
- ✓ TypeScript strict, no implicit any. Imports use the explicit `type FC` / `type KeyboardEvent` form rather than the `React.` namespace, which is consistent with the rest of the codebase.
- ✓ `closePicker` and `handleClickOutside` are wrapped in `useCallback` so `useClickOutside` doesn't re-attach document listeners on every render.
- ✓ The dead board-switcher code in `Sidebar.tsx` is fully removed (state, refs, effects, imports, and the `Star` import that's no longer needed). No leftover commented-out blocks.
- ✓ The help-FAB stacking math in `DashboardView.tsx` is now readable: `boardNavVisible`, `musicVisible`, `onLeftSide`, `leftStackSlot` named variables plus a comment documenting that each slot is 32px + 8px ≈ 2.5rem (so slot 2 = `bottom-[6.5rem]`). Much better than the old nested ternary.

### Widget Standards
✓ Not applicable — this PR doesn't touch widgets.

### Admin Config Standards
✓ Not applicable.

### Firestore & Security
✓ No rule or collection changes.

### Regression Risk
**Low**, with one architectural note:
- The music FAB is now hard-pinned to `left-4` regardless of dock position, where previously it flipped to the right side when the dock was on the left. The author addressed the gemini comment about this directly: the FAB cluster is fixed at bottom-left in all dock configurations and the music FAB stacks vertically above it, so the design choice is "all overlays on one edge" rather than "split between edges depending on dock." That's a deliberate UX call, not an oversight.
- Removing the toolbar board-switcher loses the horizontal-scroll affordance for many-board cases. The popover handles this with `overflow-y-auto` + `max-h-[60vh]`, which is functionally equivalent.
- `BoardNavFab` returns `null` when `dashboards.length <= 1`, so it won't render before the dashboards list hydrates from Firestore.

### Missing Tests
**Coverage gap.** No tests exist for `BoardNavFab.tsx`. This is a new keyboard-heavy interactive component and would benefit from at least:
- A render test confirming the cluster doesn't render with ≤1 dashboard.
- A keyboard test exercising ↑/↓/Home/End/Esc on the picker.
- A focus-management test confirming focus moves into the menu on open and back to the trigger on close.

This isn't blocking — the component is small and the keyboard handler logic is straightforward — but worth filing as a follow-up.

### Overall
**Merge readiness:** Ready with minor notes
Solid accessibility work: `role="menu"` with `aria-labelledby`, full keyboard navigation, focus-visible rings on every interactive element, focus restoration on close. The refactor of the help-FAB stacking math is a nice cleanup. The two recommendations are non-blocking: (1) consider adding a unit test for the keyboard handler before merge, (2) consider whether the deliberate "all FABs on one edge" design holds up when you also have the dock + sidebar competing for the same edge in some configurations — worth a smoke test in each dock position.

The 8 prior review comments (gemini + copilot) on accessibility, ref cleanup, useCallback wrapping, and the focus-visible ring are all addressed in the current head sha (`ad85e87`); replies have been posted to the still-open threads pointing at the relevant lines.

---
_This review was generated automatically. All findings should be verified by a human before merging._

---
_Generated by [Claude Code](https://claude.ai/code)_
